### DISCUSSION TITLE
Installation still not working :(

### DISCUSSION THREAD

**Discussion by Rhoenit** [telekom-security/tpotce#1552] [Q&A] [answered]
I still haven´t managed to install the new version. I tried using the image of the Debian version. Unfortunately I keep getting messages indicating that I need to download additional components that aren´tdocumented. For example I had to install git for the first command. Now I tried the command shown in the image but there is an issue with the Playbook. What is this and how can I proceed from here? Can someone assist me? 

![tpotproblem](https://github.com/telekom-security/tpotce/assets/78480832/c817c5f4-671a-45c5-aa04-2a7c0c546a9e)

**Comment by t3chn0m4g3** [comment]
The error indicates that another process is trying to run updates in the background which is why the installer cannot acquire an install lock. Are you using one of the recommended [distros](https://github.com/telekom-security/tpotce?tab=readme-ov-file#choose-your-distro) or running updates in a different shell or have a desktop environment i.e. Gnome active?

**Reply by Rhoenit** [reply]
I´m using ESXI and installed the recommended distro for Debian. How can I see what is active?

**Reply by t3chn0m4g3** [reply]
According to the screenshot you provided the reason is [`packagekitd`](https://wiki.debian.org/PackageKit) which usually is an indicator of a desktop environment being present. Please follow the ReadMe and install a minimal installation to avoid conflicts like that.

**Comment by Rhoenit** [comment]
I tried it again on a fresh installation with the recommended OpenSuse Tumbleweed. Still got an error when I tried: 
env bash -c "$(curl -sL https://github.com/telekom-security/tpotce/raw/master/install.sh)" 

 
![tpotpstillproblem](https://github.com/telekom-security/tpotce/assets/78480832/6c485c40-8070-4f3f-a4fa-5f18f424f510)

**Reply by t3chn0m4g3** [reply]
From your $HOME folder, provide a `ls -al` output / screenshot.

**Comment by Rhoenit** [comment]
![home](https://github.com/telekom-security/tpotce/assets/78480832/5e619129-2c66-4dcf-87f7-1aba19e06fdb)

**Reply by t3chn0m4g3** [reply]
This is not the `$HOME` folder, this is the `/home` folder. Run `ls -al $HOME`

**Reply by t3chn0m4g3** [reply, marked as answer]
Fresh install on Tumbleweed..., works perfectly fine.
![image](https://github.com/telekom-security/tpotce/assets/4318452/f5ea8d34-e6d0-45cc-be05-9f9100dfeb3c)
